pinot-common/src/main/java/org/apache/pinot/common/metadata/ZKMetadataProvider.java

```diff
@@ -131,6 +130,12 @@ public static String constructPropertyStorePathForSegmentLineage(String tableNam
   }
 
   public static String constructPropertyStorePathForMinionTaskMetadata(String taskType, String tableNameWithType) {
```

Comment:
nit: would suggest to keep the order of the params same as the order in the path for this new one (tableNameWithType, taskType)